### PR TITLE
Restructure Galley sources

### PR DESCRIPTION
**Pull request by nmittler**
This is a series of simple moves in preparation for #10995

### CODE REVIEW

File: galley/pkg/source/kube/stats/stats.go [hunk 1]
```diff
@@ -44,12 +46,12 @@ var (
 var (
 	listenerHandleEventError = stats.Int64(
 		"galley/kube/source/listener_handle_event_error_total",
-		"The number of times the listener's handleEvent has errored",
```
Comment (ozevren) [inline galley/pkg/source/kube/stats/stats.go]: Runaway search/replace?

File: galley/pkg/source/kube/stats/stats.go [hunk 2]
```diff
@@ -44,12 +46,12 @@ var (
...
+		"The number of times the source's handleEvent has errored",
 		stats.UnitDimensionless)
 
 	listenerHandleEventSuccess = stats.Int64(
 		"galley/kube/source/listener_handle_event_success_total",
-		"The number of times the listener's handleEvent has succeeded",
```
Comment (ozevren) [inline galley/pkg/source/kube/stats/stats.go]: Same

File: galley/pkg/source/kube/stats/stats.go (hunk 1 above)
Comment (nmittler) [reply galley/pkg/source/kube/stats/stats.go]: done

File: galley/pkg/source/kube/stats/stats.go (hunk 2 above)
Comment (nmittler) [reply galley/pkg/source/kube/stats/stats.go]: done.

**Comment by nmittler** [comment]
@ozevren PTAL

**Comment by nmittler** [comment]
@geeknoid can I get your approval? Just one minor change to mixer.

**Pull request merged**